widgets/show-dataviz-widget.php

```diff
@@ -67,6 +74,8 @@ public function update( $new_instance, $old_instance ) {
 
   $instance = array();
   $instance['id'] = ( ! empty( $new_instance['id'] ) ) ? strip_tags( $new_instance['id'] ) : '';
+  $instance['hide_description'] = ( ! empty( $new_instance['hide_description'] ) ) ? strip_tags( $new_instance['hide_description'] ) : "false";
```

Comment:
@DragonVirus please refer to https://github.com/OpenDevelopmentMekong/wp-odm_theme/blob/master/inc/widgets/odm-custom-posts-widget.php#L51, it should be the same implementation